Steaker/Controller/PropertiesViewController.swift

```diff
@@ -12,40 +12,36 @@ class PropertiesViewController: UITableViewController {
 
     
     @IBOutlet weak var nextButton: UIButton!
-    // private przed var o ile nie jest uzywana ta zmienna w innym Controllerze
-    /* private */ var propertiesBrain = PropertyBrain()
+    
+    private var propertiesBrain = PropertyBrain()
     
     
     override func viewDidLoad() {
         super.viewDidLoad()
 
         // Do storyboarda można wrzucić
-        nextButton.layer.masksToBounds = true
-        // To można wrzucić do metody viewDidLayoutSubviews
-        nextButton.layer.cornerRadius = nextButton.frame.size.height / 5
+        //nextButton.layer.masksToBounds = true
         
         // UITableViewController ma metodę do ustawiania wysokości komórki - patrz linijka 42
-        tableView.rowHeight = 60
+        //tableView.rowHeight = 60
```

Comment:
Zakomenotwany zbędny kod można usunąć, jak co to ja widzę co było i co jest więc luzik